### PR TITLE
38 tutorial add loss

### PR DESCRIPTION
**Pull request by QianyeYang**
# Description

Please include a summary of the change and which issue is fixed. Please also include
relevant motivation and context. List any dependencies that are required for this
change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not
      work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so
we can reproduce. Please also list any relevant details for your test configuration If
there are no tests written, flag with label so we can help generate some tests before
merge.

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by DeepRegNet** [comment]
@NMontanaBrown can you have a quick look at the failed check please?

**Comment by QianyeYang** [comment]
> 
> 
> @NMontanaBrown can you have a quick look at the failed check please?

Thanks in advance! Is the check failed only because the coverage reduced?

**Comment by mathpluscode** [comment]
the reason codecov is complaining is that, this branch didnt merge the latest master into it, i think

File: docs/tutorial_add_loss.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    y_true = tf.expand_dims(y_true, axis=4)
+    y_pred = tf.expand_dims(y_pred, axis=4)
+    if name == "lncc":
+        return -local_normalized_cross_correlation(y_true, y_pred, **kwargs)
+    elif name == "ssd":
+        return ssd(y_true, y_pred)
+    elif name == "gmi":
+        return -global_mutual_information(y_true, y_pred)
+    else:
+raise ValueError("Unknown loss type.")
```
Comment (mathpluscode) [inline docs/tutorial_add_loss.md:59]: plz correct the indent for this "raise"

File: docs/tutorial_add_loss.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+## Step 2: Add test functions (For developers, optional for users)
+
+Add correspoding unit test for the new added functions to `deepreg/test/unit`. It's just
+optional for the users. We welcome everyone who wants to make contribution to deepreg.
+Please follow our
+[contribution guidelines](https://github.com/DeepRegNet/DeepReg/blob/20-mutual-information/docs/CONTRIBUTING.md)
```
Comment (mathpluscode) [inline docs/tutorial_add_loss.md:67]: use this link

https://deepregnet.github.io/DeepReg/#/CONTRIBUTING?id=unit-tests

for the guidlines of tests

**Pull request merged**